BubbleSort.java

```diff
@@ -0,0 +1,40 @@
+
+public class BubbleSort {
+
+    public void arrayDec(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            array[i] = i + 1;
+        }
+    }
+
+    public void arrayAsc(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            array[i] = array.length - i;
+        }
+    }
+
+    public void printArray(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            System.out.print(array[i] + " ");
+        }
+    }
+
+    public void bubbleSorting(int[] array) {
+        int n = array.length;

```

Comment:
to n wydaje się niepotrzebne, można korzystać bezpośrednio z array.length

Reply:
Może i niepotrzebne ale jakie wygodne w momencie kombinowania :D